build.sbt

```diff
@@ -1375,11 +1370,15 @@ lazy val instrumentationSettings = frgaalJavaCompilerSetting ++ Seq(
 lazy val `runtime-language-epb` =
   (project in file("engine/runtime-language-epb"))
     .settings(
+      frgaalJavaCompilerSetting,
       inConfig(Compile)(truffleRunOptionsSettings),
       truffleDslSuppressWarnsSetting,
-      instrumentationSettings
+      instrumentationSettings,
+      libraryDependencies ++= Seq(
+        "org.graalvm.truffle" % "truffle-api"           % graalMavenPackagesVersion % "provided",
+        "org.graalvm.truffle" % "truffle-dsl-processor" % graalMavenPackagesVersion % "provided"
+      )
     )
-    .dependsOn(`polyglot-api`)
```

Comment:
There is no need why `EpbLanguage` shall depend on Scala & other heavyweight dependencies in the `polyglot-api` project. Test written in f87c1f6a46 shall verify some of the enormous set of dependencies of `polyglot-api` aren't present.